network-config-analyzer/K8sService.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright 2020- IBM Inc. All rights reserved
+# SPDX-License-Identifier: Apache2.0
+#
+
+from enum import Enum
+import Peer
+
+
+class K8sService:
+    """
+    Represents a K8s Service, storing its parameters
+    """
+
+    class ServiceType(Enum):
+        ClusterIP = 0  # default
+        NodePort = 1
+        LoadBalancer = 2
+        ExternalName = 3
+
+    class ServicePort:
+        """
+        Represents a K8s Service port
+        """
+        def __init__(self, port, target_port, protocol, name=''):
+            self.port = port
+            self.target_port = target_port  # a target port may be a number or a string (named port)
+            self.protocol = protocol
+            self.name = name
+
+    def __init__(self, name, namespace):
+        self.name = name
+        self.namespace = namespace
+        self.type = self.ServiceType.ClusterIP
+        self.selector = {}
+        self.ports = {}  # a map from service port name to ServicePort object
+        self.target_pods = Peer.PeerSet()
+
+    def __eq__(self, other):
+        if isinstance(other, K8sService):
+            return self.name == other.name
```

Comment:
```suggestion
            return self.name == other.name and self.namespace == other.namespace
```
services are identified by the combination of namespace + name

Reply:
Got it. Done.